src/models/mod.rs

```diff
@@ -1,2 +1,4 @@
 pub mod error;
+pub mod ids;
```

Comment:
Creating an ids file isn't a great idea. From my opinion, using the specific file, so `teams` for the team for example is more readable than just a file where you put every single id in ta single file

Reply:
I can move the individual id definitions into the relevant submodules, but then most files will have to import the ids from multiple different modules.  I would also have to keep the `ids` module for the `Base64Id` implementation.  If you're fine with that, I'll do it, but it will increase the amount of imports in every file that uses them.

Reply:
Increasing the amount of imports isn't a problem. A cleaner way is to move the base62 encoding to the mod.rs file sinc it's a utility for all submodules.